### ISSUE TITLE
Make modals accessible

### ISSUE THREAD

**Issue by neoziro** [smooth-code/smooth-ui#31]
Actually modals are not accessible for severals reasons (see #25 and #26).

Reach UI makes a good work on a11y exploration, see https://ui.reach.tech/dialog.
